docs/ops/detection/PriorBox_1.md

```diff
@@ -82,80 +116,36 @@
 
     * **Description**: *fixed_ratio* is an aspect ratio of a box. For example, *fixed_ratio* equal to 2.000000 means that the aspect ratio for the first box aspect ratio is 2.
     * **Range of values**: a list of positive floating-point numbers
-    * **Type**: `float[]`
+    * **Type**: float[]
     * **Default value**: None
```

Comment:
Not required attribute must have default value. Please go through all attributes and check correctness of `Default value`.